src/test/java/org/jenkinsci/plugins/workflow/cps/FlowDurabilityTest.java

```diff
@@ -135,7 +138,8 @@ static WorkflowRun createAndRunBasicJob(Jenkins jenkins, String jobName, FlowDur
                 "semaphore 'halt' \n" +
                 "} \n" +
                 "echo 'I like chese'\n", false);
-        def.setDurabilityHint(durabilityHint);
+        TestDurabilityHintProvider provider = Jenkins.getInstance().getExtensionList(TestDurabilityHintProvider.class).get(0);
```

Comment:
BTW `ExtensionList.lookup`; and a `TODO` comment to use `lookupSingleton` when available (later than your current baseline I think).